cypress/e2e/notes-100/select-a-note.cy.js

```diff
@@ -1,34 +1,24 @@
-import '@percy/cypress'
-import {
-  homepageSetup,
-  returningUserVisitsHomepageWaitForModel,
-} from '../../support/utils'
+import {visitHomepageWaitForModel, homepageSetup, setIsReturningUser} from '../../support/utils'
 
 
-/** {@link https://github.com/bldrs-ai/Share/issues/1055} */
-describe('Notes 100: Select a note', () => {
-  beforeEach(homepageSetup)
-  context('Returning user visits homepage', () => {
-    beforeEach(returningUserVisitsHomepageWaitForModel)
-    context('Open Notes > first note', () => {
-      beforeEach(() => {
-        cy.get('[data-testid="control-button-notes"]').click()
-        cy.get('[data-testid="list-notes"] :nth-child(1) > [data-testid="note-body"]').first().click()
-      })
-      it('Shows title, comments and new nav state', () => {
-        // The list of notes is updated to display only the selected note
-        cy.get('.MuiCardHeader-title').contains('Local issue 2')
-
-        // A list of comments attached to the note to be visible
-        cy.get('[data-testid="list-notes"] > :nth-child(2) > [data-testid="note-card"] p').contains('Test Comment 1')
-        cy.get('[data-testid="list-notes"] > :nth-child(3) > [data-testid="note-card"] p').contains('Test Comment 2')
-
-        cy.get('[data-testid="panelTitle"]').should('have.text', 'NOTE')
-
-        cy.get('[data-testid="Back to the list"]')
-
-        cy.percySnapshot()
-      })
+describe('select-a-note', () => {
+  context('Open index.ifc and notes', () => {
+    beforeEach(() => {
+      homepageSetup()
+      setIsReturningUser()
+      visitHomepageWaitForModel()
+    })
+    it('The list of notes is updated to display only the selected note and list of comments is visible', () => {
+      cy.get('[data-testid="control-button-notes"]').click()
+      cy.get(':nth-child(1) > [data-testid="note-card"] > [data-testid="note-body"] > .MuiCardContent-root').contains('issueBody_4').click()
+      cy.get('.MuiCardHeader-title').contains('issueTitle_4')
+      cy.get(':nth-child(2) > [data-testid="note-card"] > .MuiCardContent-root > p').contains('testComment_1')
+    })
+    it('The title on the navbar changes to NOTE and back button visible', () => {
+      cy.get('[data-testid="control-button-notes"]').click()
+      cy.get(':nth-child(1) > [data-testid="note-card"] > [data-testid="note-body"] > .MuiCardContent-root').contains('issueBody_4').click()
+      cy.get('.css-1oum0wi > .css-8lgfcg > :nth-child(1) > .css-95g4uk > [data-testid="panelTitle"]').should('have.text', 'NOTE')
```

Comment:
I know we talked about not using the CSS selectors. But for some reason I could not get it to pass otherwise. looking into it further.